packages/superset-ui-chart-controls/src/shared-controls/index.tsx

```diff
@@ -481,7 +481,8 @@ const sharedControls = {
   time_range,
   row_limit,
   limit,
-  timeseries_limit_metric: enableExploreDnd ? dnd_timeseries_limit_metric : timeseries_limit_metric,
+  timeseries_limit_metric: enableExploreDnd ? dnd_sort_by : sort_by,
+  orderby: enableExploreDnd ? dnd_sort_by : sort_by,
```

Comment:
Since this control relates to ordering by metric (as opposed to ordering by e.g. a column name), perhaps we should rename `orderby` to `orderby_metric`?

Reply:
no problem at all.

But this control is just for declaring the orderby clause in the SQL, and I guess if we add dimension(column) sorting, we'll probably do it in this control too.

Reply:
Good points - if we don't think there will be a need for separate ordering controls for 1) only columns 2) only metrics 3) columns and metrics, then this is probably the right solution for now 👍